### DISCUSSION TITLE
No-convergence in the FIR

### DISCUSSION THREAD

**Discussion by getrude386** [mboquien/cigale#72] [Q&A]
Hi, can you please assist with figuring out the reason why there is no convergence in the FIR? I have attached my SED for one of the sources, input data, and the pcigale.ini file.
[80sources.txt](https://github.com/mboquien/cigale/files/13598327/80sources.txt)

![Screenshot 2023-12-07 at 11 42 21](https://github.com/mboquien/cigale/assets/105718721/681063d6-268c-4bb6-946f-a835833ad7ba)
[pcigale (1).txt](https://github.com/mboquien/cigale/files/13598336/pcigale.1.txt)

**Comment by mboquien** [comment]
The flux at the shortest wavelength is not consistent with the rest of the spectrum. The five fluxes at longer wavelengths are also inconsistent. This make it impossible to find a good fit.

**Reply by getrude386** [reply]
Ok, I will look into that thank you.

Also, since I have added the radio model and radio photometry too, Was I not supposed to obtain a synchrontron emission line? Also because it is mentioned on the legend.

**Reply by Guang91** [reply]
Your radio fluxes seem to be higher than the model. Is it a radio loud AGN? If so, you may consider including higher values of R_agn.

**Reply by getrude386** [reply]
Yes, it's a radio-loud AGN. Thank you I will work on that.

**Comment by ghost** [comment]
Hey, I was just curious where did you get your flux values for the VIRCAM data? And also the more parameters I add to the modules I am using the more time it is taking. I don't know if it is because I am using google colab or what? It is taking way too long to run is there any way I can reduce the time because it taking more than an hour.

**Reply by getrude386** [reply]
Hi, I am using a catalog compiled by someone else, but this paper "https://www.aanda.org/articles/aa/pdf/2012/08/aa19507-12.pdf" can provide information on where or how to obtain the catalog.

**Comment by ghost** [comment]
heyy, thank you so much for writing back. Will definitely look into it.
Thanks again !!!
